react-design/my-app/src/RotatingBanner.js

```diff
@@ -0,0 +1,42 @@
+import NavigationButton from './NavigationButton';
+import IndexButtons from './IndexButtons';
+import Banner from './Banner';
+import { useState } from 'react';
+
+export default function RotatingBanner({ items }) {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  function handleButtonsOnClick(e) {
+    const int = parseInt(e.target.textContent);
+    setCurrentIndex(int);
+  }
```

Comment:
I generally recommend against passing full event objects up the component tree. Instead, only pass the value the parent component needs (in this case, the index).